manifests/1.14/rbac/attacher-rbac.yaml

```diff
@@ -1,39 +0,0 @@
-apiVersion: v1
```

Comment:
One more question I had... Did you run into problems launching the attacher and provisioner separately? They were split out with our 1.14 manifests so that they could have different service accounts and RBAC roles attached them. The CSI driver itself specifically had very limited permissions to do anything with the K8s API. Looks like all the permissions for everything have been re-combined into one big role, which to me violates the idea of least-based privilege.

Reply:
We started with creating 4 service accounts 1 for each pod attacher, provisioner, controller, and metadata syncer.

Later while deploying them on the setup having multiple master nodes, we realized all these pods are getting deployed randomly on multiple master nodes. 

We want to make sure attacher, provisioner and controller remains in a single node, as they communicate over Unix domain socket. If they are on different nodes, the system does not work.

so to ensure guarantee, these containers run on a single node, we embedded all of them in a single pod.

Only one service account can be assigned to a pod, so we merged all roles created for these individual service accounts and assigned them to a single service account `vsphere-csi-controller`.